include/cpl/data_structure/lazyprop_segtree.hpp

```diff
@@ -70,9 +85,9 @@ class lazyprop_segtree {
   /// \par Complexity
   /// Linear in <tt>count</tt>.
   ///
-  lazyprop_segtree(size_t count, const T &identity,
-                   const Combine &comb = Combine())
-      : num_elems{count}, combine(comb) {
+  lazyprop_segtree(size_t count, const T& identity,
+                   const Combine& comb = Combine())
```

Comment:
Correct me if I'm wrong but the signature of should not  be `lazyprop_segtree(size_t, const T&,Combine&&)`